robot_smach_states/src/robot_smach_states/manipulation/grab.py

```diff
@@ -217,11 +217,11 @@ def execute(self, userdata=None) -> str:
         # Remove pose from ED as we are holding the object in the gripper
         self.robot.ed.update_entity(uuid=grab_entity.uuid, remove_pose=True)
 
-        arm.wait_for_motion_done(cancel=True)
-
```

Comment:
While it often wont change the behaviour, I do think its important to keep this check